cmd/api/book/service.go

```diff
@@ -37,23 +37,50 @@ func (s *Service) ArchiveBook(id uuid.UUID) (Book, error) {
 	return s.repo.SetBookArchiveStatus(id, archived)
 }
 
-func (s *Service) CreateBook(bookEntry Book) (Book, error) {
-	bookEntry.ID = uuid.New()                                      //Atribute an ID to the entry
-	bookEntry.CreatedAt = time.Now().UTC().Round(time.Millisecond) //Atribute creating and updating time to the new entry. UpdateAt can change later.
-	bookEntry.UpdatedAt = bookEntry.CreatedAt
-	return s.repo.CreateBook(bookEntry)
+type EntryBookRequest struct {
+	Name      string
+	Price     *float32
+	Inventory *int
+}
+
+func (s *Service) CreateBook(bookEntry EntryBookRequest) (Book, error) {
+	createdAt := time.Now().UTC().Round(time.Millisecond) //Atribute creating and updating time to the new entry. UpdateAt can change later.
+	newBook := Book{
+		ID:        uuid.New(), //Atribute an ID to the entry
+		Name:      bookEntry.Name,
+		Price:     bookEntry.Price,
+		Inventory: bookEntry.Inventory,
+		CreatedAt: createdAt,
+		UpdatedAt: createdAt,
+		//Archived is set to false by defalut inside database
```

Comment:
Achive is `false` by default since the zero value of a boolean is false.

Reply:
Yes, but is set at the migrations, too: 
```
ALTER TABLE public.bookstable
  ADD COLUMN IF NOT EXISTS archived BOOLEAN DEFAULT false;
```
I've put this comment just to keep in mind that this field is present too, although it's not being initialized in this function. Do you think should I take it off?

Reply:
well... there is no harm in the comment, just think that if you add such a comment you may start to rely on them.
Then when you forget to add it for some reason and it will make your life harder.